src/bylines/index.js

```diff
@@ -53,10 +63,13 @@ const BylinesSettingsPanel = () => {
 				contentEl.insertBefore( bylineEl, contentEl.firstChild );
 			}
 			// If there are author tags
-			if ( /<Author id=(\d+)>/.test( text ) ) {
-				text = text.replace( /<Author id=(\d+)>([^<]+)<\/Author>/g, ( match, authorId, authorName ) => {
-					return `<a href="${ newspackBylines.siteUrl }/?author=${ authorId }">${ authorName }</a>`;
-				} );
+			if ( /\[Author id=(\d+)\]/.test( text ) ) {
+				text = text.replace(
+					/\[Author id=(\d+)\]([^\[]+)\[\/Author\]/g,
```

Comment:
Non-blocking: Similar to above, this could use a code comment to clarify what we are doing here.

Reply:
This is all being completely rewritten in https://github.com/Automattic/newspack-plugin/pull/3746